images/base-python-adapter/swagger_server/swagger/swagger.yaml

```diff
@@ -85,7 +85,7 @@ paths:
   /adapterDefinition:
     get:
       summary: Retrieve the data model for collection
-      description: Get Definition
```

Comment:
I had to re-generate the server to add the extra parameters to the AdapterConfig. It turns out that there had been some changes in the input swagger files since that had happened last - most of these changes are just reflecting that but don't affect anything in the server.

The important changes for this PR are on lines 509-514 and 1113-1123

Reply:
This file itself is auto-generated from `vmware_aria_operations_integration_sdk/api/vmware-aria-operations-collector-fwk2.json` and `vmware_aria_operations_integration_sdk/api/integration-sdk-definition-endpoint.json` using the script `images/update-base-python-adapter-server.py`